### PR TITLE
feat: Add getNodes() and addNodes() methods.

### LINKED ISSUE #21: Add ability to persist and load nodes
Continuing https://github.com/hyperswarm/dht/issues/4.

### PR DESCRIPTION
**Pull request by aureooms**
Fixes #21.

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -246,6 +246,61 @@ tape('persistent', function (t) {
...
+    a.command('hello', {
+      query (data, callback) {
+        t.same(data.value, null, 'expected data')
+        callback(null, Buffer.from('world'))
+      }
+    })
+
+    a.ready(function () {
+      b.ready(function () {
+        setTimeout(function () {
```
Comment (mafintosh) [inline test.js:283]: does process.nextTick work here or does it have to be a setTimeout?

Comment (aureooms) [reply test.js:283]: I considered trying that but did not because the docs said `nextTick` gets priority over the rest of the queue.

Comment (mafintosh) [reply test.js:283]: Can you try? I think this is because the ready handler is called before the node is added.

Comment (aureooms) [reply test.js:283]: It works! What should I do?

Comment (mafintosh) [reply test.js:283]: Nice! Replace setTimeout with that then. That much easier to reason about test wise :)

Comment (aureooms) [reply test.js]: Done!

**Comment by aureooms** [comment]
Something else to test for?

**Comment by mafintosh** [comment]
Looks good, I’ll merge today

**Comment by mafintosh** [comment]
4.9.0, thank you for that

**Comment by mafintosh** [comment]
I realise now we didn't add it to the docs, so we should do that at some point

**Comment by mafintosh** [comment]
@aureooms Unrelated, I just noticed it says you are based in Copenhagen on your profile. Most of the hyper* team is here as well.

If you want to, shoot me an email at mathiasbuus@gmail.com, maybe we can meet up some time and hack/discuss.

**Comment by aureooms** [comment]
@mafintosh Will do!

**Comment by aureooms** [comment]
> I realise now we didn't add it to the docs, so we should do that at some point

I sent PR #23 for that!

**Pull request merged**